src/inspector_agent.cc

```diff
@@ -711,8 +739,11 @@ void Url(const FunctionCallbackInfo<Value>& args) {
 void Agent::InitInspector(Local<Object> target, Local<Value> unused,
                           Local<Context> context, void* priv) {
   Environment* env = Environment::GetCurrent(context);
+  env->set_inspector_console_api_object(Object::New(env->isolate()));
```

Comment:
Just seems to be the standard practice for passing objects into C++... but feel free to do whatever you prefer.